packages/next/client/index.js

```diff
@@ -180,7 +180,9 @@ async function doRender ({ App, Component, props, err, emitter: emitterProp = em
   // In development runtime errors are caught by react-error-overlay.
   if (process.env.NODE_ENV === 'development') {
     renderReactElement((
-      <App {...appProps} />
+      <RouterContext.Provider value={makePublicRouterInstance(router)}>
```

Comment:
Ok, so I think this is a big breaking change for `next-graphql-react`, and probably the Apollo examples too:

https://github.com/jaydenseric/next-graphql-react/blob/v1.0.1/src/index.mjs#L92

Reply:
Are we sure this new approach of `<RouterContext.Provider>` wrapping `<App>` is here to stay? If so, I can work it in to `next-graphql-react` and cut a new release.

Reply:
The way @alexandernanberg is exactly how I would have implemented it, outside of _app.js.